src/bundle/Resources/public/admin/js/ibexa-automated-translation.js

```diff
@@ -1,57 +1,61 @@
-jQuery(function () {
-    let $ = jQuery;
-    let $form = $("form[name=add-translation]", "#add-translation-modal");
-    let $container = $(".ibexa-automated-translation-services-container:first", $form);
-    let $error = $(".ibexa-automated-translation-error", $container);
-
-    $form.click(function () {
-        $error.addClass("invisible");
-    });
-
-    $container.find(".ibexa-field-edit--ezboolean .ibexa-data-source__label").click(function () {
-        let $input = $(this).find("input[type='checkbox']");
-        let isChecked = $input.attr('checked') === 'checked';
-
-        if (isChecked) {
-            $input.removeAttr('checked');
-            $(this).removeClass('is-checked');
+(function (global, doc) {
+    const modal = doc.querySelector('#add-translation-modal');
+    const form = modal.querySelector('form[name="add-translation"]');
+    const container = form.querySelector('.ibexa-automated-translation-services-container');
+    const error = container.querySelector('.ibexa-automated-translation-error');
+    const sourceLabel = container.querySelector('.ibexa-field-edit--ezboolean .ibexa-data-source__label');
+
+    const toggleClass = (event) => {
+        const target = event.target;
+        const input = target.querySelector('input[type="checkbox"]');
```

Comment:
Are there any ibexa classes on this input? If yes, then it probably should be sth like (at least that's class that SHOULD be on checkbox input ;) )
```suggestion
        const input = target.querySelector('.ibexa-input--checkbox');
```